diff --git a/PWG/EMCAL/macros/AddTaskEmcalTriggerMaker.C b/PWG/EMCAL/macros/AddTaskEmcalTriggerMaker.C
index ce43da2..286935b 100644
--- a/PWG/EMCAL/macros/AddTaskEmcalTriggerMaker.C
+++ b/PWG/EMCAL/macros/AddTaskEmcalTriggerMaker.C
@@ -31,7 +31,7 @@ AliEmcalTriggerMaker* AddTaskEmcalTriggerMaker(
     return NULL;
   }
 
-  TString strTrigersName(triggersName);
+  TString strTriggersName(triggersName);
   TString strCellsName(cellsName);
 
   if(strTriggersName.IsNull()) {
@@ -45,7 +45,7 @@ AliEmcalTriggerMaker* AddTaskEmcalTriggerMaker(
     }
   }
 
-  if(strCellssName.IsNull()) {
+  if(strCellsName.IsNull()) {
     if (evhand->InheritsFrom("AliESDInputHandler")) {
       strCellsName = "EMCALCells";
       ::Info("AddTaskEmcalTriggerMaker", Form( "ESD analysis, cellsName = \"%s\"", strCellsName.Data() ));
